src/client/app/components/MenuModalComponent.tsx

```diff
@@ -60,7 +60,7 @@ export default class MenuModalComponent extends React.Component<MenuModalProps,
 							<p style={labelStyle}>
 								<FormattedMessage id='navigation' />:
 							</p>
-							<HeaderButtonsContainer  showCollapsedMenuButton={false} />
+							<HeaderButtonsContainer  showCollapsedMenuButton={false} isModal />
```

Comment:
This may well be my lack of knowledge about this so just let me know. I'm confused about isModal in two ways. First, it does not seem to be set in this file. Is it assuming a default? Second, it is supplying this value the HeaderButtonsContainer but that does not seem to be part of its props. However, it all seems to work great.